.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Hardhat Ignition",
+    "image": "mcr.microsoft.com/devcontainers/base:bullseye",
+    "features": {
+        "ghcr.io/devcontainers/features/node:1": {
+          "version": "16" /* Keep in sync with the oldest version of Node.js that Hardhat supports */
```

Comment:
We should bump this to node 18.

Node 16 EOL got brought forward: https://nodejs.org/en/blog/announcements/nodejs16-eol

Our supported Node.js versions are those in Active LTS or maintenance LTS + 2 months after they leave maintenance LTS.

Node 16 has been 2 months out of maintenance LTS as of a few weeks ago.